### PR TITLE
2020-05-07 learning team meeting

### PR DESCRIPTION
**Pull request by mansona**

### CODE REVIEW

File: learning-team/2020-05/2020-05-07.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- [x]  Ricard Mendes (RM)
+- [x]  Jessica Jordan (JJ)
+- [ ]  Sivakumar Kailasam (SK)
+- [ ]  Melanie Sumner (MS)
+- [ ]  Jen Weber (JW)
+- [ ]  Kenneth Larsen (KL)
+- [ ]  Robert Wagner (RW)
+
+### Guests
+
```
Comment (Gaurav0) [inline learning-team/2020-05/2020-05-07.md:21]:
```suggestion

 - [x]  Gaurav Munjal

```

Comment (alexeykostevich) [inline learning-team/2020-05/2020-05-07.md:21]: - [x]  Alexey Kostevich (AK)

Comment (alexeykostevich) [inline learning-team/2020-05/2020-05-07.md:21]:
```suggestion
- [x]  Alexey Kostevich (AK)
```

Comment (amyrlam) [reply learning-team/2020-05/2020-05-07.md:21]: Also:
Jared Galanis @jaredgalanis 
Frédéric Soumaré @hakilebara

**Pull request merged**